### PR TITLE
Create component TableRow

### PR DESCRIPTION
**Pull request by thanhtrantruong47**
Create component TableRow

image

### CODE REVIEW

File: examples/practice-typescript/package-lock.json
Comment (hungdo-agilityio) [inline examples/practice-typescript/package-lock.json:1]: Why this file change? Did u use `npm` or `pnpm`?

File: examples/practice-typescript/src/components/TableRow/index.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+}
+
+const TableRow = ({
+  id,
+  email,
+  firstName,
+  lastName,
+  phoneNumber,
+}: TableRowProps) => {
+  return (
```
Comment (hungdo-agilityio) [inline examples/practice-typescript/src/components/TableRow/index.tsx]: shorthand

File: examples/practice-typescript/src/scripts/app.tsx [hunk 2]
```diff
@@ -50,6 +51,22 @@ const App = () => {
...
+          lastName="kim"
+          phoneNumber="0132456789"
+        />
+         <TableRow
+          id="2"
+          email="kim@gmail.com"
+          firstName="tran "
+          lastName="kim"
+          phoneNumber="0132456789"
+        />
```
Comment (hungdo-agilityio) [inline examples/practice-typescript/src/scripts/app.tsx]: using map rendering

File: examples/practice-typescript/src/components/TableRow/index.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    <tr key={id}>
+      <td className="table-item">{id}</td>
+      <td className="table-item">{email}</td>
+      <td className="table-item">{firstName}</td>
+      <td className="table-item">{lastName}</td>
+      <td className="table-item">{phoneNumber}</td>
+      <td className="table-item" style={{ display: 'flex', gap: '40px' }}>
```
Comment (hungdo-agilityio) [inline examples/practice-typescript/src/components/TableRow/index.tsx]: why inline style here

File: examples/practice-typescript/src/components/TableRow/index.tsx [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+        <Button text="Edit" styleClass="btn-action" />
+        <Button text="Delete" styleClass="btn-action" />
```
Comment (hungdo-agilityio) [inline examples/practice-typescript/src/components/TableRow/index.tsx]: props for these action?

File: examples/practice-typescript/src/mocks/user.ts [hunk 5]
```diff
@@ -0,0 +1,9 @@
+export const User = [
```
Comment (hungdo-agilityio) [inline examples/practice-typescript/src/mocks/user.ts]: missing type

File: examples/practice-typescript/src/components/TableRow/index.tsx (hunk 1 above)
Comment (thanhtrantruong47) [reply examples/practice-typescript/src/components/TableRow/index.tsx]: Fixed

File: examples/practice-typescript/src/scripts/app.tsx (hunk 2 above)
Comment (thanhtrantruong47) [reply examples/practice-typescript/src/scripts/app.tsx]: Fixed and I using map rendering in next PR TableBody

File: examples/practice-typescript/src/components/TableRow/index.tsx (hunk 3 above)
Comment (thanhtrantruong47) [reply examples/practice-typescript/src/components/TableRow/index.tsx]: Fixed

File: examples/practice-typescript/src/mocks/user.ts (hunk 5 above)
Comment (thanhtrantruong47) [reply examples/practice-typescript/src/mocks/user.ts]: Fixed

File: examples/practice-typescript/src/components/TableRow/index.tsx (hunk 4 above)
Comment (thanhtrantruong47) [reply examples/practice-typescript/src/components/TableRow/index.tsx]: Fixed

File: examples/practice-typescript/package-lock.json
Comment (thanhtrantruong47) [reply examples/practice-typescript/package-lock.json:1]: I added the package but deleted it afterwards.

File: examples/practice-typescript/src/components/TableRow/index.tsx [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+  id,
+  email,
+  firstName,
+  lastName,
+  phoneNumber,
+  onEdit,
+  onDelete,
+}: TableRowProps) => (
+  <tr key={id}>
+    <td className="table-item">{id}</td>
```
Comment (thanhnguyen-agilityio) [inline examples/practice-typescript/src/components/TableRow/index.tsx]: `table-cell` or `row-item`

File: examples/practice-typescript/src/scripts/app.tsx [hunk 7]
```diff
@@ -5,6 +5,14 @@ import SideBarMenu from 'components/SideBarMenu';
 import TableHead from 'components/TableHead';
 import { MenuItems } from 'mocks/menu';
 import { USER_TABLE_HEADERS, PRODUCT_TABLE_HEADERS } from './constants/user';
+import TableRow from 'components/TableRow';
+
+const editUser = () => {
+  console.log('edit');
+};
```
Comment (thanhnguyen-agilityio) [inline examples/practice-typescript/src/scripts/app.tsx]: add one line break

File: examples/practice-typescript/src/components/TableRow/index.tsx (hunk 6 above)
Comment (thanhtrantruong47) [reply examples/practice-typescript/src/components/TableRow/index.tsx]: Fixed

File: examples/practice-typescript/src/scripts/app.tsx (hunk 7 above)
Comment (thanhtrantruong47) [reply examples/practice-typescript/src/scripts/app.tsx]: Fixed

**Review by thanhnguyen-agilityio — APPROVED** [review]

**Pull request merged**